shell-completion/zsh/_systemd-run

```diff
@@ -57,4 +57,6 @@ _arguments \
         '--on-unit-inactive=[Run after SEC seconds from the last deactivation]:SEC' \
         '--on-calendar=[Realtime timer]:SPEC' \
         '--timer-property=[Set timer unit property]:NAME=VALUE' \
+        '--timeout=[Set maximum time for the service to run]:timeout' \
+        '--wait=[Wait until service stopped again]' \
```

Comment:
Would be nice to have a separate commit for adding the recent `--wait` to the completion. (Same for the bash completion above).